service-mesh/README.md

```diff
@@ -0,0 +1,7 @@
+# Service Mesh E-Commerce Example
+
+This folder contains an example app that can be deployed and monitored on a
+service mesh. Click on your mesh of choice for an example deployment. The app
+source is viewable at [app-src](./app-src).
```

Comment:
Any chance you'd be able to link to some outside resources, maybe w/ the istio project at minimum?